test/integration/test-amp-bind.js

```diff
@@ -39,25 +39,28 @@ describe.configure().skipEdge().run('amp-bind', function() {
     /* eslint-enable max-len */
     extensions: ['amp-bind'],
   }, env => {
-    let doc, text;
+    let browser, doc, text;
+
+    this.timeout(TIMEOUT);
```

Comment:
Feels weird having this just sit out here. Can this be put into a `before()`?

Reply:
I think [hook level timeouts](https://mochajs.org/#hook-level) apply to the hook only, whereas this is intentionally a [suite level timeout](https://mochajs.org/#suite-level). But per our offline discussion, I think it makes sense to pull this out one level higher. The reason for increasing the timeout is due to the service worker and bind-evaluator XHR load being slow. This would be a global problem for all of the tests in `amp-bind`, but probably not surface since the first flaky test that I've updated in this PR runs first (if it fails nothing else will run).